### DISCUSSION TITLE
Figure shortcode not Applying Class

### DISCUSSION THREAD

**Discussion by keesfluitman** [nunocoracao/blowfish#1481] [General]
Hey Just a question, 
Love your work thanks!

I have added this shortcode
{{< figure
    src="/gallery/image.jpg"
    alt="image"
    class=".customSize"
    >}}
  But the class is not applied, and my custom.css file with the class thus not working.
  Any other way to add images and control their position?

As a workaround I just add html and element css directly. But maybe I'm overlooking something?

**Comment by nunocoracao** [comment]
This is related to https://github.com/nunocoracao/blowfish/issues/1112 there is a bug currently opened.
